src/chatClient.cpp

```diff
@@ -1852,6 +1855,7 @@ promise::Promise<void> GroupChatRoom::decryptTitle()
         wptr.throwIfDeleted();
         KR_LOG_ERROR("Error decrypting chat title for chat %s:\n%s\nFalling back to member names.", karere::Id(chatid()).toString().c_str(), err.what());
         makeTitleFromMemberNames();
+        return err;
```

Comment:
I suggest to manage the failed promise by the caller. If the decryption of title, as you said, fails... it makes sense to return the error (despite the title will be set to the peer's names).